test/utils/goHelpers/cmd/receiptMMPTProof/main.go

```
@@ -0,0 +1,87 @@
+	if err := trieFun.Prove(rlp.AppendUint64(nil, logIdx), &targetReceiptProof); err != nil {
+		fmt.Println("error proving receipt: ", err)
+		return
+	}
+
+	for key, value := range targetReceiptProof {
+		fmt.Println("key:", hex.EncodeToString([]byte(key)))
+		fmt.Println("value:", hex.EncodeToString(value))
+	}
+
+	var buf bytes.Buffer
+	receipts[0].EncodeRLP(&buf)
```

golangci-lint (1.61.0)

Error return value of `(*github.com/ethereum/go-ethereum/core/types.Receipt).EncodeRLP` is not checked

(errcheck)